jetson/nav/search/searchStateMachine.cpp

```diff
@@ -224,8 +311,15 @@ NavState SearchStateMachine::executeTurnToTarget()
         }
         return NavState::TurnToTarget;
     }
+    double gimbalAddition = 0.0;
+    if ( mRoverConfig[ "search" ][ "useGimbal" ].GetBool() ){
+        gimbalAddition = mRover->gimbal().getYaw();
+        mRover->gimbal().setDesiredGimbalYaw(mRover->roverStatus().leftCacheTarget().bearing);
+        mRover->publishGimbal();
+    }
     if( mRover->turn( mRover->roverStatus().leftCacheTarget().bearing +
-                      mRover->roverStatus().odometry().bearing_deg ) )
+                      mRover->roverStatus().odometry().bearing_deg +
+                      gimbalAddition ) )
```

Comment:
why is the bearing offset added here but not in the gate version?

Reply:
wait why is the odometry bearing being used? also are these even all in the same coordinate frame??????